firedantic/_async/model.py

```diff
@@ -38,14 +38,14 @@
 }
 
 
-def get_collection_name(cls, name: Optional[str]) -> str:
+def get_collection_name(cls, name: Optional[str] = None) -> str:
     if not name:
         raise CollectionNotDefined(f"Missing collection name for {cls.__name__}")
 
     return f"{CONFIGURATIONS['prefix']}{name}"
 
 
-def _get_col_ref(cls, name: Optional[str]) -> AsyncCollectionReference:
+def _get_col_ref(cls, name: Optional[str] = None) -> AsyncCollectionReference:
```

Comment:
These don't seem like changes we want to do?

Reply:
I agree. I don't see why we would want to have a default value for these parameters. End users are not really supposed to call them directly and if they would, it's not really helpful at all to have a default value that will cause it to raise and exception. The reason the parameter is annotated as optional is that it's possible the user has accidentally forgot to specify the collection name in the model, causing it to default to None.